scapy/data.py

```diff
@@ -306,7 +304,7 @@ def _process_data(fdesc):
                 line = line.strip()
                 if not line:
                     continue
-                lt = tuple(re.split(spaces, line))
```

Comment:
Did you notice anything having removed this? I wonder if this was not here for some weird compatibility stuff

Reply:
I think the new code is equivalent, but a bit cleaner. Am I wrong?

Reply:
To my understanding you're right. I just wonder why `\n` was in the regex. Good for me